theano/scalar/basic.py

```diff
@@ -3843,7 +3843,7 @@ def __init__(self, inputs, outputs):
         self.nin = len(inputs)
         self.nout = len(outputs)
         self.init_fgraph()       # self.fgraph
-
+        self.init_c_code()
```

Comment:
remove that. This is not the right fix. I think this is an old work around and we have the right fix in master. So if after removing, you still have problems, tell us.